src/PolyhedralGeometry/Cone/properties.jl

```diff
@@ -486,6 +534,29 @@ _generator_matrix(::Val{_hilbert_generator}, C::Polymake.BigObject; homogenized=
 _matrix_for_polymake(::Val{_hilbert_generator}) = _generator_matrix
 
 
+@doc raw"""
+    contains(C0::Cone, C1::Cone)                           
+    
+Check whether `C0` contains `C1`.
```

Comment:
Contains in which sense? As a set, as a subcone, or something else?

Reply:
A subset, I clarified.